src/analysis/modifications/gene_associations.jl

```diff
@@ -0,0 +1,52 @@
+
+"""
+    change_eflux_constraints(
```

Comment:
Should probably call it `add_eflux_constraints` to fit in with all the other mods (some also change constraints)

Reply:
yeah I played with this. Will change to add.